src/index.js

```diff
@@ -75,9 +105,66 @@ class CreateDynamoDBGlobalTables {
     }
   }
 
+  async tableIsAtVersionNumber(tableName) {
+    const dynamo = new AWS.DynamoDB({ region: MASTER_REGION });
+    const params = {
+      TableName: tableName,
+    };
+
+    let globalTableVersion = '';
+
+    try {
+      const tableDetails = await dynamo.describeTable(params).promise();
+      globalTableVersion = tableDetails.Table.GlobalTableVersion;
+    } catch (error) {
+      this.log(error);
+    }
+
+    this.log(`Table ${tableName} is at version ${globalTableVersion}`);
+
+    return globalTableVersion === GLOBAL_TABLE_VERSION;
+  }
+
+  async updateReplicaVersion(tableName, region) {
+    this.log(`Checking replica version for ${tableName}`);
+    const canUpdate = await this.tableIsAtVersionNumber(tableName);
```

Comment:
looks like this only checks the master region and not the region that's passed as an argument?